drafts/build_a_dao_on_fuel/draft.md

```diff
@@ -0,0 +1,1416 @@
+[note]: <> ("note to self and file editor, there are specific comments left throughout this file, for the sake of understanding the author's reason for using a particular method of approach; please feel free to delete once acknowledged or reply edit with a better approach 🥂")
+
+# How to Build a DAO on the Fuel blockchain.
+
+## Introduction
+
+A Decentralized Autonomous Organization (DAO) is considered an essential blockchain application.
+
+It is a decentralized application that users or organizations use to automate the governance execution of their projects and communities. The automated program is represented by rules encoded in a network of computers that is transparently controlled by the organization members and not influenced by any central authority.
+
+By completing this article, you will learn how to create a Dao contract on the Fuel blockchain. You will also gain knowledge about the following concepts:
+
+1. Writing a smart contract using the sway language.
+2. Understanding how to use low-level code fundamentals in sway `asm`, `call`, etc.
```

Comment:
Make sure "Sway" is always capitalized, i.e. "Sway" not sway

Reply:
Thanks a lot, I've made all changes in the recent commit